src/components/groupImpactComponents/__tests__/GroupImpactSidebar.test.js

```diff
@@ -115,7 +121,7 @@ describe.skip('GroupImpactSidebar component', () => {
 
     expect(wrapper.find(Slide).prop('in')).toEqual(false)
     expect(gtag).toHaveBeenCalledWith('event', 'group_impact_sidebar', {
-      interaction: 'open',
+      interaction: 'close',
```

Comment:
Some out of dates were failing because we had originally turned off tests on this component. This was due to the memory issues we had about 1-2mo ago.